statsmodels/tsa/statespace/_tools.pyx.in

```diff
@@ -765,10 +843,10 @@ cdef int _{{prefix}}reorder_missing_rows({{cython_type}} * a, int * missing, int
         nobs = nobs - missing[i]
 
     # Perform replacement
-    k = nobs-1
-    for i in range(n-1,-1,-1):
+    k = nobs - 1
+    for i in range(n - 1, -1, -1):
         if not missing[i]:
-            blas.{{prefix}}swap(&m, &a[i], &n, &a[k], &n)
+            swap(m, &a[i], n, &a[k], n)
             k = k - 1
```

Comment:
Note that with this done, `_{{prefix}}reorder_missing_rows` no longer depends on `{{prefix}}`, so this function in turn can be simplified to a single implementation.  Most of this module can be boiled down to single implementations.